emcc.py

```diff
@@ -3243,9 +3243,10 @@ def phase_embind_emit_tsd(options, in_wasm, wasm_target, memfile, js_syms):
   # import names.
   settings.MINIFY_WASM_IMPORTED_MODULES = False
   setup_environment_settings()
-  # Replace embind with the TypeScript generation version.
-  embind_index = settings.JS_LIBRARIES.index('embind/embind.js')
-  settings.JS_LIBRARIES[embind_index] = 'embind/embind_ts.js'
+  # Ensure embind is included at least once.
+  settings.JS_LIBRARIES.index('embind/embind.js')
```

Comment:
Is this supposed to be like an assert?

Why not be more explict? : `assert 'embind/embind.js' in settings.JS_LIBRARIES`.. then you don't need the comment I think.